fastplotlib/utils/add_graphics.py

```diff
@@ -0,0 +1,537 @@
+from typing import *
+
+import numpy
+
+from fastplotlib.graphics import *
```

Comment:
use relative import

Reply:
when I use relative import I get this error:

`ImportError: attempted relative import with no known parent package`

Reply:
Even with relative import, I get errors that the graphics are not being imported

so then when I have the return type defined, I am getting import errors that the graphic names are note defined which will also impact the instantiation in each of the methods of a given graphic

Reply:
You probably need to go up two levels, from ..graphics

Reply:
Paste the errors

Reply:
```
  File "/home/clewis7/repos/fastplotlib/fastplotlib/layouts/generate_add_methods.py", line 3, in <module>
    from ..graphics import *
ImportError: attempted relative import with no known parent package
```